packages/sdk/typescript/human-protocol-sdk/test/escrow.test.ts

```diff
@@ -1071,9 +1076,12 @@ describe('EscrowClient', () => {
     });
 
     test('should successfully cancel escrow', async () => {
+      mockTx.wait.mockResolvedValueOnce({ transactionHash: FAKE_HASH });
+
       const escrowAddress = ethers.constants.AddressZero;
 
       escrowClient.escrowFactoryContract.hasEscrow.mockReturnValue(true);
+      escrowClient.escrowContract.cancel.mockResolvedValueOnce(mockTx);
```

Comment:
Check if the hash returned by the `cancel` method is the same than `FAKE_HASH`

Reply:
Updated